src/managers/GuildChannelManager.js

```diff
@@ -79,8 +79,8 @@ class GuildChannelManager extends CachedManager {
   /**
    * Options used to create a new channel in a guild.
    * @typedef {Object} GuildChannelCreateOptions
-   * @property {string} [type='text'] The type of the new channel, either `text`, `voice`, `category`, `news`,
-   * `store`, or `stage`
+   * @property {string} [type='GUILD_TEXT'] The type of the new channel, either `GUILD_TEXT`, `GUILD_VOICE`,
```

Comment:
While you're at it, this should take a number, I'd much like to be able to pass in ChannelTypes.GUILD_TEXT

Reply:
I don't think that's possible due to this line:
https://github.com/discordjs/discord.js/blob/ab0b3b9a07f5e4987e4f25e41b2a007f2db06322/src/managers/GuildChannelManager.js#L131
Unless you want me to change that as well?